gem/lib/pupperware/spec_helper.rb

```diff
@@ -263,8 +263,8 @@ def get_container_network(container)
   def emit_log(container)
     container_name = get_container_name(container)
     STDOUT.puts("#{'*' * 80}\nContainer logs for #{container_name} / #{container}\n#{'*' * 80}\n")
-    logs = run_command("docker logs --details --timestamps #{container}")[:stdout]
-    STDOUT.puts(logs)
+    # run_command streams stdout / stderr
+    run_command("docker logs --details --timestamps #{container}" 2>&1)[:stdout]
```

Comment:
Whoa.

The `2>&1` isn't within the `"..."` quotes?

I assume that `2>&1` works on Windows?

Reply:
Ha nope! Just a typo it looks like based on the Azure failure 😄

Reply:
Haha, yeah... I slapped this together heading out the door. No surprise that I typo'd it!